airflow/config_templates/config.yml

```
@@ -452,6 +452,14 @@ core:
 database:
   description: ~
   options:
+    alembic_ini_file_path:
+      description: |
+        Path to a custom alembic.ini file. If provided, these configurations
+        will be prioritised over the default alembic.ini file.
```

I think the description needs to be updated
```suggestion
        Path to the ``alembic.ini`` file. You can either provide the file path relative 
        to the Airflow home directory or the absolute path if it is located elsewhere.
```

---

Yes, that definitely makes more sense.